spec/lib/pretty_text_spec.rb

```diff
@@ -2143,11 +2139,11 @@ def expect_cooked_match(raw, expected_cooked)
     expect(cooked).to eq(html)
   end
 
-  it "provides safety for img bbcode" do
-    cooked = PrettyText.cook "[img]http://aaa.com<script>alert(1);</script>[/img]"
-    html =
-      '<p><img src="http://aaa.com&lt;script&gt;alert(1);&lt;/script&gt;" alt="" role="presentation"></p>'
-    expect(cooked).to eq(html)
+  it "supports img bbcode entities in attributes" do
```

Comment:
Note: it isn't so much that we support it, but I would like to keep evidence in our codebase that this is not an issue.